tests/test_helpers.py

```diff
@@ -27,7 +27,8 @@
      "2006as_timbuktoo has been added in expenses\n"),
     (["helper", "a_presentation", "Mars", "flat earth", "--type", "contributed_oral",
       "--begin_date", "2020-06-26", "--end_date", "2020-06-26", "--person", "ashaaban",
-    "--authors", "sbillinge", "--abstract", "the earth is round as seen from mars", "--title", "On the roundness of the Earth"],
```

Comment:
let's give a list of authors as a better test.   Does this test capture all the optional args?  Best if the test can cover more rather than less, so make sure that all optional args are triggered (except the boilerplate ones like ``--database``) .

Reply:
done